# this mgs stores the x, y coordinates of the box along with width and height
float32 x
float32 y
float32 w
float32 h
float32 score
